### PR TITLE
Create bio for chotai

### PR DESCRIPTION
**Pull request by chotai**
I'm a teacher's pet!

@githubstudent!! #34

### CODE REVIEW

File: something-awesome/something-awesomer/bio [hunk 1]
```diff
@@ -0,0 +1 @@
+This is my awesome cv!
```
Comment (chotai) [inline something-awesome/something-awesomer/bio:1]: wow this really is awesome!

Comment (chotai) [inline something-awesome/something-awesomer/bio:1]: could add some more epic information!

**Comment by alcala21** [comment]
:+1:

**Comment by ChuaMichelle** [comment]
@dmellenthin bop :100:

**Pull request merged**